modules/ml/src/main/java/org/apache/ignite/ml/math/functions/IgniteFunction.java

```diff
@@ -34,7 +34,37 @@
      * @param <R> Type of output.
      * @return {@link IgniteFunction} returning specified constant.
      */
-    static <T, R> IgniteFunction<T, R> constant(R r) {
-        return t -> r;
+    public static <T, R> IgniteFunction<T, R> constant(R r) {
+        return new IgniteFunction<T, R>() {
+            /** {@inheritDoc} */
+            @Override public String toString() {
+                return "Constant function [c=" + r + "]";
```

Comment:
it looks good but we don't use toString for functions)

Reply:
Yes, intention was to make debugging less painful with at least some of the lambdas listed with meaningful names, but maybe we should make a ticket for that.